gsa/src/web/pages/audits/__tests__/table.js

```diff
@@ -0,0 +1,278 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+import React from 'react';
+
+import Capabilities from 'gmp/capabilities/capabilities';
+import CollectionCounts from 'gmp/collection/collectioncounts';
+import {setLocale} from 'gmp/locale/lang';
+
+import Filter from 'gmp/models/filter';
+import Audit, {AUDIT_STATUS} from 'gmp/models/audit';
+
+import {setUsername} from 'web/store/usersettings/actions';
+
+import {rendererWith, fireEvent} from 'web/utils/testing';
+
+import Table from '../table';
+
+setLocale('en');
+
+const caps = new Capabilities(['everything']);
+
+const lastReport = {
+  report: {
+    _id: '1234',
+    timestamp: '2019-08-10T12:51:27Z',
+    compliance_count: {yes: 4, no: 3, incomplete: 1},
+  },
+};
+
+const lastReport2 = {
+  report: {
+    _id: '1234',
+    timestamp: '2019-07-10T12:51:27Z',
+    compliance_count: {yes: 4, no: 3, incomplete: 1},
+  },
+};
+
+const currentReport = {
+  report: {
+    _id: '5678',
+    timestamp: '2019-07-10T12:51:27Z',
+  },
+};
+
+const audit = new Audit({
+  _id: '1234',
+  owner: {name: 'admin'},
+  name: 'foo',
+  comment: 'bar',
+  status: AUDIT_STATUS.done,
+  alterable: '0',
+  last_report: lastReport,
+  permissions: {permission: [{name: 'everything'}]},
+  target: {_id: 'id1', name: 'target1'},
+});
+
+const audit2 = new Audit({
+  _id: '12345',
+  owner: {name: 'user'},
+  name: 'lorem',
+  comment: 'ipsum',
+  status: AUDIT_STATUS.new,
+  alterable: '0',
+  permissions: {permission: [{name: 'everything'}]},
+  target: {_id: 'id2', name: 'target2'},
+});
+
+const audit3 = new Audit({
+  _id: '123456',
+  owner: {name: 'user'},
+  name: 'hello',
+  comment: 'world',
+  status: AUDIT_STATUS.running,
+  alterable: '0',
+  current_report: currentReport,
+  last_report: lastReport2,
+  permissions: {permission: [{name: 'everything'}]},
+  target: {_id: 'id2', name: 'target2'},
+});
+
+const counts = new CollectionCounts({
+  first: 1,
+  all: 1,
+  filtered: 1,
+  length: 1,
+  rows: 2,
+});
+
+const filter = Filter.fromString('rows=2');
+
+describe('Table tests', () => {
```

Comment:
Could you name your tests a little more specific so that we can easily see which table is tested in comparison to the policy table for example. I'm thinking about simply naming this one "Audits table test". When yarn runs all the tests it's faster to see what went wrong